ci-operator/step-registry/ci-operator/step-registry/openstack-k8s-operators/test/build-deploy/openstack-k8s-operators-test-build-deploy-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: openstack-k8s-operators-test-build-deploy
+  steps:
+    pre:
+    - ref: ipi-install-rbac
```

Comment:
to make things simple, you can maintain only one main step here: "openstack-k8s-operators-deploy" for instance.